docs/pages/attributes.rst

```diff
@@ -0,0 +1,215 @@
+
+ 	<objectname>.phys_properties.json
+
+Object Mesh Handles And Types
+-----------------------------	
+
+Below are the handles and descriptors for various mesh assets used by an object.
+	
+Mesh types are included to support future format flexibility. In the JSON they are limited to string keys used by :ref:`AbstractPhysicsAttributes::AssetTypeNamesMap`, which map to values in :ref:`AssetType`.  Any mesh type specification used in JSON but not found in :ref:`AbstractPhysicsAttributes::AssetTypeNamesMap` results in that mesh type being set to :ref:`AssetType::UNKNOWN`.  JSON-Specified mesh types will override any default settings, such as those that might be derived from file name extensions.
+
+
+"render mesh" 
+	- string
+	- The name of the file describing the render mesh to be used by the object.
+"render mesh type"
+	- string
+	- The type of the render mesh. If not specified, defaults to :ref:`AssetType::UNKNOWN`.
+"collision mesh"
+	- string
+	- The name of the file describing the collision mesh to be used by the object. 
+"collision mesh type"
+	- string
+	- The type of the collision mesh. If not specified, defaults to :ref:`AssetType::UNKNOWN`.
+	
+
+Object Frame and Origin
+-----------------------
+
+The tags below are used to build a coordinate frame for the object, and will override any default values set based on render mesh file name/extension.  If either **"up"** or **"front"** are specified, both must be provided and they must be orthogonal.  The object's COM is used as its origin.
+
+"up"
+	- 3-vector
+	- Describes the **up** direction for the object.
+"front"
+	- 3-vector
+	- Describes the **forward** direction for the scene.
+
+
+Below are object-specific physical quantities.  These values will override similarly-named values specified in a Scene Attributes or Physics Manager Attributes.
+
+"scale"
+	- 3-vector
+	- The default scale to be used for the object.
+"mass"
+	- double
+	- The mass of the object, for physics calculations.
+"margin"
+	- double
+	- Distance margin for collision calculations.
+"friction coefficient"
+	- double
+	- The coefficient of friction.
+"restitution coefficient"
+	- double
+	- The coefficient of restitution.
+"units to meters"
+	- double
+	- The conversion of given units to meters.
+"requires lighting"
+	- boolean
+	- Whether or not the scene should use lighting.
+"inertia"
+  - 3-vector
+  - The values of the diagonal of the inertia matrix for the object.
+"COM"
+  - 3-vector
+  - The center of mass for the object.  If this is not specified in JSON, it will be derived from the object's bounding box in Habitat.
+"use bounding box for collision"
+  - boolean
+  - Whether collision calculations should be based on object's bounding box geometry.
+"join collision meshes"
+  - boolean
+  - Whether collision mesh assets should be joined into a single unified collision object.
```

Comment:
How should a user choose this? Is there any reason *not* to join collision meshes?

Reply:
This is a field that I inherited and so I am not certain what it's purpose is, but, just shooting in the dark if there are multiple meshes used to construct an object then joining them _may_ be less efficient for collision calcs - many small bounding boxes vs a single large one.